components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/consent/SSOConsentServiceImpl.java

```diff
@@ -402,7 +404,7 @@ private Receipt getConsentReceiptOfUser(ServiceProvider serviceProvider, Authent
             }
         } catch (ConsentManagementException e) {
             throw new SSOConsentServiceException("Consent Management Error",
-                                                 "Error while retrieving user consents.", e);
+                    "Error while retrieving user consents.", e);
```

Comment:
Shall we format the line by moving the 2nd parameter to the above line? 
```
            throw new SSOConsentServiceException("Consent Management Error", "Error while retrieving user consents.", 
                    e);
```

Reply:
Fixed in 06271985d0d22192422630521503f04db25e5eff.